src/mscorlib/shared/System/IO/DisableMediaInsertionPrompt.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.IO
+{
+    /// <summary>
+    /// Simple wrapper to safely disable the normal media insertion prompt for
+    /// removable media (floppies, cds, memory cards, etc.)
+    /// </summary>
+    /// <remarks>
+    /// Note that removable media file systems lazily load. After starting the OS
+    /// they won't be loaded until you have media in the drive- and as such the
+    /// prompt won't happen. You have to have had media in at least once to get
+    /// the file system to load and then have removed it.
+    /// </remarks>
+    internal sealed class DisableMediaInsertionPrompt : IDisposable
```

Comment:
Since structs do not have parameter-less constructors, you need `Create` method instead. Otherwise. It should be the same as with class:

```
internal struct DisableMediaInsertionPrompt : IDisposable
{
    public static DisableMediaInsertionPrompt Create()
    {
    }

    public void Dispose()
    {
        ....
    }        
}

...
using (DisableMediaInsertionPrompt.Create())
{
    ...
}
...
```